### PR TITLE
Migrate unit/test_utils.py from unittest to pytest

### PR DESCRIPTION
**Pull request by AndresSan6**
*Issue #, if available:* #99

*Description of changes:*
Migrated "test/unit/test_utils.py" from unittest to pytest

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Comment by mattsb42-aws** [comment]
Pull that `six` import out to pass the linting checks, and looks good.

File: test/unit/test_utils.py [hunk 1]
```diff
@@ -139,49 +138,43 @@ def setUp(self):
         )
         self.mock_aws_encryption_sdk_instance.decrypt.return_value = VALUES["data_key"]
         self.mock_aws_encryption_sdk_instance.encrypt.return_value = VALUES["encrypted_data_key"]
-
-    def tearDown(self):
```
Comment (mattsb42-aws) [inline test/unit/test_utils.py]: Missing a `yield`.

Comment (AndresSan6) [reply test/unit/test_utils.py]: I wrote it in the commit message but didn't add it to the code, my bad. I already added it in the last commit.

Comment (mattsb42-aws) [reply test/unit/test_utils.py]: Yup, saw that commit come through right as I published that comment. 👍

**Pull request merged**